RoddyCore/src/de/dkfz/roddy/core/RuntimeService.groovy

```diff
@@ -608,7 +668,94 @@ public abstract class RuntimeService extends CacheProvider {
 
     public abstract Map<String, Object> getDefaultJobParameters(ExecutionContext context, String TOOLID)
 
-    public abstract String createJobName(ExecutionContext executionContext, BaseFile file, String TOOLID, boolean reduceLevel)
+    public String createJobName(ExecutionContext executionContext, BaseFile file, String TOOLID, boolean reduceLevel) {
+        return JobManager.getInstance().createJobName(file, TOOLID, reduceLevel);
```

Comment:
The advantag ist, that you can have job names set on a workflow level rather than job manager specific.